bin/jar

```diff
@@ -1 +1 @@
-.openjdk@17.pkg
\ No newline at end of file
+.openjdk@21.pkg
```

Comment:
Generally it's not advisable to use channel packages if you want reproducible builds. Up to you though

Reply:
Ah I think I understand what you are saying here, thanks for catching.